src/cluster/slot_migrate.cc

```diff
@@ -382,6 +382,12 @@ Status SlotMigrator::sendSnapshotByCmd() {
     }
   }
 
+  if (auto s = iter->status(); !s.ok()) {
+    auto err_str = s.ToString();
+    LOG(ERROR) << "[migrate] Failed to iterate keys of slot " << slot << ": " << err_str;
+    return {Status::NotOK, fmt::format("failed to iterate keys of slot {}: {}", slot, s.ToString())};
```

Comment:
Can also use err_str here.

Reply:
Hi @git-hulk ,

Thanks for your suggestion!
I have replaced `s.ToString()` with `err_str`.

Best Regards,
Edward